### ISSUE TITLE
Feature request: Options to autoactivate the charging station on power on and to prevent deactivation after charging

### ISSUE THREAD

**Issue by FrenkK** [CurtRod/SimpleEVSE-WiFi#16]
Hello :) First of all, thank you for this very useful project, it fills a real need!
I'm using it on my home charging station for three primary purposes: for monitoring, to be able to dynamically adjust charging amperage to prevent my main fuses from blowing out (using the setCurrent API) and for the power measurement capability. Configuration: a S0 energy meter, no RFID, no button.
I have two issues:
1. Currently, the controller will turn off the SimpleEVSE after each charge (writing 8192 to register 2005 at the start). This is a problem on my Ampera for two reasons:
  - The car stops indicating a full charge and I can't verify a proper charge without turning the car back on. We have very unstable power, so charging problems happen more often than you would think.
  - Ampera should be always connected to a charger in extremely warm or freezing weather so the system can protect the battery by heating or cooling it (AFAIK, this goes for other e-cars as well).
2. There are several of us using this station and everybody expects to just plug in the cable and leave. If a  short blackout occurs (and it does quite frequently), the station restarts, but does not reactivate, so they have to first check if it is charging and then activate it, if it doesn't, This already caused some failed charges.

I think these issues will probably be bothering other people as well, so it would be nice to have two simple checkboxes in config: "Do not deactivate after finished charge" and "Automatically activate the station on power on".

**Comment by Leftaf** [comment]
Hi Frenkk,

For the handling with the Cars there is a possibility to improve. If you have a cable with Plug normally PP is fix with GND connected e.g. 680 ohm to indicate 16A . If you modify the Plug to that the PP to GND can be opened. The Car would normally get reactivated and will try to charge again. 

To avoid blackouts on the esp6288 side you may use the latest version with battery support. [WEMOS V2.0.0](https://wiki.wemos.cc/products:d1:d1_mini_pro)

**Comment by CurtRod** [comment]
@FrenkK Thank you for that feature request. I have already considered that this may be a possible scenario. Via hardware it will probably not be possible to stop setting register 2005. Each time the ESP is restarted, EVSE WB is automatically deactivated.
The problem with the implementation right now is that it's not so easy to notice that a charge has been started. This would require the EVSE WB to be polled for every few seconds. But it's not impossible. I'll include that in the backlog as a new feature. At the moment, however, there is still a stability issue with the HTTP API, which should be fixed before. I keep that in focus.

**Comment by FrenkK** [comment]
Thank you for keeping this in mind. I actually bodged up a quick fix to prevent the autodeactivate on startup and recompiled, but it is not even close to a proper solution.

**Comment by ronmichel** [comment]
@FrenkK Could you be so kind and share your quick fix to prevent the autodeactivate on startup? Thanks so much!

**Comment by FrenkK** [comment]
@ronmichel What I did was to strip all the code I don't need from the 2.6 version and then add the logic that keeps it on. There is also a tweak to the JSON reporting of current, I removed the unnecessary nesting in the result that was messing with my Home Assistant setup.
You are free to use the slimmed down code attached, but it's a quick and dirty job with some functions (like Modbus meter support) removed.
[SimpleEVSE-WiFi.zip](https://github.com/CurtRod/SimpleEVSE-WiFi/files/3700710/SimpleEVSE-WiFi.zip)
